checkov/cdk/checks/python/S3IgnorePublicACLs.yaml

```diff
@@ -0,0 +1,14 @@
+metadata:
+  version: 0.2
+  approach: define failing
+  id: CKV_AWS_55
+  name: Ensure S3 bucket has ignore public ACLs enabled
+  category: GENERAL_SECURITY
+scope:
+  languages:
+    - python
+definition:
+  pattern: aws_cdk.aws_s3.Bucket(<ANY>, block_public_access=$ARG,<ANY>)
```

Comment:
1. No need to split the `$ARG` pattern into the metavariable condition, as it all can be just a single line pattern (instead of the `$ARG` you can put here the `aws_cdk.aws_s3.BlockPublicAccess(ignore_public_acls=False)`
2. What about a default s3 bucket with no param of `block_public_access`? isn't it implicitly violating the policy as well?

Reply:
You're right. Adjusted.